.github/workflows/ci.yml

```diff
@@ -509,7 +480,7 @@ jobs:
         if: always()
         with:
           # Name prefix is checked in the `Annotate checks` workflow
-          name: test report ${{ github.job }}
+          name: test report ${{ github.job }} (${{ env.ARTIFACT_NAME }})
```

Comment:
wouldn't this render as `test report test (module path)`?

Can we change to just `report (module path)`?

Reply:
For product tests use `report pt (suite, config)`?

Reply:
This is out of the scope of this PR.